docker/docker-compose.yml

```diff
@@ -0,0 +1,55 @@
+database:
+  image: "quay.io/galexrt/zulip-postgresql-tsearchextras:latest"
+  environment:
+    DB_NAME: zulip
+    DB_USER: zulip
+    DB_PASS: zulip
+  volumes:
+    - "/opt/docker/zulip/postgresql/data:/var/lib/postgresql:rw"
+memcached:
+  image: "quay.io/sameersbn/memcached:latest"
+  restart: always
+rabbitmq:
+  image: "docker.io/rabbitmq:3.5.5"
```

Comment:
Docker 1.7.1

Seems like it is only happening on the docker-compose up

```
$ docker-compose up
Creating dockerzulip_camo_1...
Creating dockerzulip_database_1...
Creating dockerzulip_memcached_1...
Creating dockerzulip_redis_1...
Pulling rabbitmq (docker.io/rabbitmq:3.5.5)...
Traceback (most recent call last):
  File "<string>", line 3, in <module>
  File "/compose/build/docker-compose/out00-PYZ.pyz/compose.cli.main", line 31, in main
  File "/compose/build/docker-compose/out00-PYZ.pyz/compose.cli.docopt_command", line 21, in sys_dispatch
  File "/compose/build/docker-compose/out00-PYZ.pyz/compose.cli.command", line 34, in dispatch
  File "/compose/build/docker-compose/out00-PYZ.pyz/compose.cli.docopt_command", line 24, in dispatch
  File "/compose/build/docker-compose/out00-PYZ.pyz/compose.cli.command", line 66, in perform_command
  File "/compose/build/docker-compose/out00-PYZ.pyz/compose.cli.main", line 460, in up
  File "/compose/build/docker-compose/out00-PYZ.pyz/compose.project", line 230, in up
  File "/compose/build/docker-compose/out00-PYZ.pyz/compose.service", line 320, in execute_convergence_plan
  File "/compose/build/docker-compose/out00-PYZ.pyz/compose.service", line 208, in create_container
  File "/compose/build/docker-compose/out00-PYZ.pyz/compose.service", line 236, in ensure_image_exists
  File "/compose/build/docker-compose/out00-PYZ.pyz/compose.service", line 693, in pull
  File "/compose/build/docker-compose/out00-PYZ.pyz/docker.client", line 883, in pull
  File "/compose/build/docker-compose/out00-PYZ.pyz/docker.auth.auth", line 59, in resolve_repository_name
  File "/compose/build/docker-compose/out00-PYZ.pyz/docker.auth.auth", line 38, in expand_registry_url
docker.errors.DockerException: HTTPS endpoint unresponsive and insecure mode isn't enabled.
```

`$ docker run docker.io/rabbitmq:3.5.5` does seem to work and will pull and run, and then `docker-compose up` works fine once it has been pulled down once.